index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/4339e938ba.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>my portofolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <header class="logo">Amanuel</header>
+        <a href="#" class="humburger"><i class="fa-solid fa-bars"></i></a>
+      </nav>
```

Comment:
Hi @amanuelbelete95, Please change the `nav` tag to a `header` because it will make your HTML more semantic.
You can put the `nav` tag inside the `header` if you want.

For more info please [check this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)